src/app/(mobile-ui)/points/page.tsx

```diff
@@ -1,32 +1,115 @@
 'use client'
 
-import Icon from '@/components/Global/Icon'
+import { Button } from '@/components/0_Bruddle'
+import PageContainer from '@/components/0_Bruddle/PageContainer'
+import Card, { getCardPosition } from '@/components/Global/Card'
+import CopyToClipboardButton from '@/components/Global/CopyToClipboard/CopyToClipboardButton'
+import { Icon } from '@/components/Global/Icons/Icon'
+import NavHeader from '@/components/Global/NavHeader'
+import PeanutLoading from '@/components/Global/PeanutLoading'
+import ShareButton from '@/components/Global/ShareButton'
+import TransactionAvatarBadge from '@/components/TransactionDetails/TransactionAvatarBadge'
+import { VerifiedUserLabel } from '@/components/UserHeader'
 import { useAuth } from '@/context/authContext'
-import peanutClub from '@/assets/peanut/peanut-club.png'
+import { invitesApi } from '@/services/invites'
+import { useQuery } from '@tanstack/react-query'
+import { useRouter } from 'next/navigation'
 
 const PointsPage = () => {
+    const { data: invites, isLoading } = useQuery({
+        queryKey: ['invites'],
+        queryFn: () => invitesApi.getInvites(),
+    })
```

Comment:
nit: invites is not specific key to user

```suggestion
const { data: invites } = useQuery({
  queryKey: ['invites', user?.user.userId],  // per-user scoped
  queryFn: () => invitesApi.getInvites(),
  enabled: !!user?.user.userId,
})
```